ShoppingList/src/main/java/org/openintents/shopping/ui/widget/ShoppingItemsView.java

```diff
@@ -88,6 +88,7 @@
     private final static boolean debug = false;
 
     private Typeface mCurrentTypeface;
+    private ShoppingActivity mShoppingActivity;
```

Comment:
Could you please use an interface here?

Reply:
Couldn't find an interface that provides onModeChanged(). I'm not that deeply into the architecture of your app.